### ISSUE TITLE
Types Parsing

### ISSUE THREAD

**Issue by 3timeslazy** [mununki/gqlmerge#3]
Hi! At first, thank for your job!

Secondly, gqlmerge falls on the follow schema:
```graphql
schema {
    query: Query
}
	
type Query {
    # first place it falls.
    # the message is `unexpected "=", expected "Ident"`
    admin(id: ID!, role: Role = ADMIN): Admin!
    user(id: ID!): User!
    search(text: String!): [SearchResult]!
}
	
interface Admin {
    id: ID!
    name: String!
    role: Role!
}

scalar Time	

type User implements Admin {
    id: ID!
    name: String!
    email: String!
    role: Role!
    phone: String!
    address: [String!]
    # second place it falls.
    # the message is `unexpected "(", expected ":"`
    friends(page: Pagination): [User]
    createdAt: Time!	
}

input Pagination {
    first: Int
    last: Int
}
	
enum Role {
    ADMIN
    USER
}

union SearchResult = User
```

**Comment by ghost** [comment]
Can you add some examples to act as tests please. Love the project but need examples to fix bugs I feel

**Comment by mununki** [comment]
@3timeslazy I've released a PR #4, please try again with it.
Thank you for your feedback.

Please let me know if you have still any issues related or I'll close this issue.

**Comment by 3timeslazy** [comment]
It works! Thank you!
